### ISSUE TITLE
TotpUrlError now defined in a private module

### ISSUE THREAD

**Issue by alexanderkjall** [constantoine/totp-rs#29]
Hi

I just tried upgrading to the 3.0.0 release, and the TotpUrlError enum seems to have moved to the url_error module and the reexport here is not public https://github.com/constantoine/totp-rs/blob/master/src/lib.rs#L55.

This gives me these kind of errors in my code:
```
error[E0603]: module `url_error` is private
  --> src/error.rs:27:27
   |
27 |     TotpUrlError(totp_rs::url_error::TotpUrlError),
   |                           ^^^^^^^^^ private module
   |
note: the module `url_error` is defined here
  --> /home/capitol/.cargo/registry/src/github.com-1ecc6299db9ec823/totp-rs-3.0.0/src/lib.rs:52:1
   |
52 | mod url_error;
   | ^^^^^^^^^^^^^^
```

or

```
error[E0603]: enum `TotpUrlError` is private
  --> src/error.rs:27:27
   |
27 |     TotpUrlError(totp_rs::TotpUrlError),
   |                           ^^^^^^^^^^^^ private enum
   |
note: the enum `TotpUrlError` is defined here
  --> /home/capitol/.cargo/registry/src/github.com-1ecc6299db9ec823/totp-rs-3.0.0/src/lib.rs:55:5
   |
55 | use url_error::TotpUrlError;
   |     ^^^^^^^^^^^^^^^^^^^^^^^
```

Am I missing something on how this is intended to be used?

**Comment by constantoine** [comment]
Hi!

I'm most sorry, this is definetly a typo. I `pub`'ed `use base32` instead of `use url_error::TotpUrlError`

I am fixing it and releasing it ASAP

**Comment by constantoine** [comment]
This should be fixed with [v3.0.1](https://github.com/constantoine/totp-rs/releases/tag/v3.0.1)

**Comment by alexanderkjall** [comment]
Wow, thanks for the super fast response :)

3.0.1 worked without problems.
